ogre_mesh_viewer.py

```diff
@@ -313,6 +313,63 @@ def preRenderTargetUpdate(self, evt):
             entity.getSubEntities()[highlight].setMaterial(self.app.highlight_mat)
             self.highlighted = highlight
 
+        if ImGui.CollapsingHeader("Transform"):
+            flags = ImGui.TableFlags_Borders | ImGui.TableFlags_SizingStretchProp
+
+            enode = entity.getParentSceneNode()
+
+            ImGui.Text("Position")
+            if ImGui.BeginTable("Position", 3, flags):
+                ImGui.TableSetupColumn("x")
+                ImGui.TableSetupColumn("y")
+                ImGui.TableSetupColumn("z")
+                ImGui.TableHeadersRow()
+                ImGui.TableNextRow()
+                ImGui.TableNextColumn()
+                ImGui.Text(str(round(enode.getPosition().x, 2)))
+                ImGui.TableNextColumn()
+                ImGui.Text(str(round(enode.getPosition().y, 2)))
+                ImGui.TableNextColumn()
+                ImGui.Text(str(round(enode.getPosition().z, 2)))
+                ImGui.EndTable()
+
+            ImGui.Separator()
+
+            ImGui.Text("Orientation")
+            if ImGui.BeginTable("Orientation", 4, flags):
+                ImGui.TableSetupColumn("w")
```

Comment:
I dont think that seeing the raw quaternion values is very helpful.
You should rather print pitch, yaw, roll as those are the rotations around local x,y,z axes.